esmvalcore/_config/extra_facets/native6-ipsl-cm6-mappings.yml

```diff
@@ -0,0 +1,256 @@
+# Mapping, for ISPLCM output formats 'Analyse' and 'Output', between a
+# CMOR variable name and the labels to use by ESMValTool to find the
+# corresponding file, and the corresponding variable in the file
+#
+# For format 'Analyse', the config-development.yml file tells
+# ESMValTool to use key 'ipsl_varname' for building the filename,
+# while for format 'Output' it specifies to use key 'group'
+#
+# Specifying 'igcm_dir' here allows to avoid having to secifiy it in
+# datasets definitions
+#
+# Key 'use_cdo' allows to choose whether CDO will be invoked for
+# selecting a variable in a multi-variable file. This generally allows
+# for smaller overal load time. But because CDO has a licence which is
+# not compliant with ESMValtool licence policy, the default
+# configuration is to avoid using it. You may use customized settings
+# by installing a modified version of this file as
+# ~/.esmvatlool/variable_details/native6-ipsl-cm6-*.yml
+#
+# Keys scale and offset can be used for handling any unit issue that
+# is not dealt wih automatically by the udunits layer in ESMValTool
+# (and which makes use of the units metadaat in input files). In that
+# case, scaling and offseting are applied automatically by method
+# fix_data of class AllVars in cmor/_fixes/*/ipsl.py
```

Comment:
I would recommend including the correct units of the variable in this file instead of using a scale factor and offset. That way the automatic unit conversion will be able to do its work. This is more reliable and will avoid problems if the unit in the CMOR table would change.

Reply:
As discussed on zoom, for flipping the sign I would recommend just providing the `postive: up` or `postive: down` attribute (if it's not available in the netcdf file already) and then writing some code that flips the sign and the attribute if it's the opposite from what is in the CMOR table.

Reply:
Done